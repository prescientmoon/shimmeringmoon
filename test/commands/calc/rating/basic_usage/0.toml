reply = true
content = "The score 9'349'070 on Arcana Eden [PRS] yields a rating of 8.20"
embeds = []
attachments = []
