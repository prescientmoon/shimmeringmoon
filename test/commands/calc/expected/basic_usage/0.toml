reply = true
content = "The expected score for a player of potential 12.27 on Vicious [ANTi] Heroism [BYD] is 9'834'000"
embeds = []
attachments = []
